TrustKit/TSKPinningValidator.h

```diff
@@ -63,6 +41,17 @@ typedef NS_ENUM(NSInteger, TSKTrustDecision)
 /// @name Manual SSL Pinning Validation
 ///------------------------------------
 
+@property (nonatomic, readonly, nullable) NSDictionary *pinnedDomains;
```

Comment:
I added some docs. We should probably convert to full models instead of a dictionary.